### DISCUSSION TITLE
Actualizaciones de repositorio falladas

### DISCUSSION THREAD

**Discussion by ghost** [sebasvelasco353/FDP-2022_01#121] [Show and tell]
![image](https://user-images.githubusercontent.com/98999407/159107382-e31edd62-cc22-4d1c-9c1b-bf82306be88d.png)

Hola profe, estoy tratando de arreglar los errores que aparecen en el repositorio pero al tratar de corregirlo como me lo dice github, me aparece luego un error de fatal... Pero como lo tienes protegido al pareccer no me deja. Sin embargo ahí aparece uno que al parecer si está funcionando.

Me puedes confirmar si funciona o no? Gracias

**Comment by ghost** [comment]
Por otro lado, a la clase de hoy no pude ir porque tuve una emergencia familiar con mi abuela y nos toco que llevarla a urgencias. Una compañera me aviso lo que habias dicho en la clase después cuando pude contestar. Muchas gracias por el plazo y que pena los inconvenientes

**Reply by sebasvelasco353** [reply]
Todavía tienes merge conflicts, eso Renés que resolverlo en tu proyecto, no con comandos de git.

Revisa bien cuando hagas merge de todo te debe aparecer el conflicto y te toca acomodar en el código, ayer revisamos en clase y tus merge conflicts son en carpetas que no son tuyas, lo cual es extraño porque no deberían de tener cambios.

La forma de resolver un merge conflicts es haciendo update a todo el codigo y revisar en los archivos cuál debe quedar: local o remoto.
